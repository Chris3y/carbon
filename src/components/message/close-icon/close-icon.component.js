import React from 'react';
import PropTypes from 'prop-types';
import CloseIconStyle from './close-icon.style';
import Icon from '../../icon/icon';

const CloseIcon = ({ as, onDismiss }) => {
  return (
    <CloseIconStyle type={ as }>
      <Icon
        data-element='dismiss' onClick={ onDismiss }
        type='close'
      />
    </CloseIconStyle>
  );
};

CloseIcon.defaultProps = {
  as: 'info'
};

CloseIcon.propTypes = {
  as: PropTypes.oneOf(['error', 'success', 'info', 'warning']),
  onDismiss: PropTypes.func
};

export default CloseIcon;
