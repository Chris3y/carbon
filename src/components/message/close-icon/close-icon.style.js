import styled from 'styled-components';
import PropTypes from 'prop-types';
import BaseTheme from '../../../style/themes/base';
import OptionsHelper from '../../../utils/helpers/options-helper';
import CloseIconClassicStyling from './close-icon-classic.style';
import Link from '../../link';

const CloseIconStyle = styled.div`
  align-items: center;
  display: flex;
  margin-left: auto;
  margin-right: 15px;
  justify-content: center;
  text-align: center;
  border: none;
  color: ${({ messageType, theme }) => theme.colors[messageType]}};

  ${CloseIconClassicStyling}
`;

const LinkStyle = styled(Link)`
  .carbon-link__content {
    display: none;
  }
  .carbon-link__icon {
    margin-right: 0;
  }

  &:focus {
    outline: none;
    span span {
      &:before {
        outline: 2px solid ${({ theme }) => theme.colors.focus};
        outline-offset: 3px;
      }
    }
  }
`;

CloseIconStyle.defaultProps = {
  messageType: 'info',
  roundedCorners: true,
  theme: BaseTheme,
  transparent: false
};

CloseIconStyle.propTypes = {
  messageType: PropTypes.oneOf(OptionsHelper.messages),
  border: PropTypes.bool,
  roundedCorners: PropTypes.bool,
  transparent: PropTypes.bool
};

LinkStyle.defaultProps = {
  theme: BaseTheme
};

export { CloseIconStyle, LinkStyle };
