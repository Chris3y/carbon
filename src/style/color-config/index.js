// Base colors
export default {
  brilliantGreen: '00DC00',
  errorRed: 'C7384F',
  gold: 'FFB500',
  genericGreen: '009900',
  infoBlue: '0077C8',
  productGreen: '00A376',
  productBlue: '0077C8',
  navyBlue: '004B87',
  amethyst: '582C83',
  plum: '8246AF',
  slate: '003349',
  successGreen: '00B000',
  warningOrange: 'E96400'
};
