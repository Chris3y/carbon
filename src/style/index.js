import generatePalette from './palette';
import atOpacity from './utils/at_opacity';

const blackAtOpacity = atOpacity('#FFFFFF');
const addOpacity = (color, opacity) => atOpacity(color)(opacity);

export {
  addOpacity,
  blackAtOpacity,
  generatePalette
};
