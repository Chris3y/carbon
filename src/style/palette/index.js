import tint from '../utils/tint';
import shade from '../utils/shade';

export default (colors) => {
  return Object.keys(colors).reduce((acc, col) => {
    const {
      base, tints, shades
    } = colors[col];

    const tintBy = tint(base);
    const shadeBy = shade(base);

    const tintedColors = tints.reduce((tintAcc, weight) => (
      { ...tintAcc, [`${col}Tint${weight}`]: tintBy(weight) }
    ), {});

    const shadedColors = shades.reduce((tintAcc, weight) => (
      { ...tintAcc, [`${col}Shade${weight}`]: shadeBy(weight) }
    ), {});

    return { ...acc, ...tintedColors, ...shadedColors, [`${col}Base`]: base };
  }, {});
};
