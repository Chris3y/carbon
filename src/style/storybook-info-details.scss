.storybook-info {
  h1 {
    border-bottom: 1px solid rgb(238, 238, 238);
    font-size: 25px;
    margin: 20px 0px 0px;
    padding: 0px 0px 5px;
  }

  code,
  .storybook-code {
    background-color: rgb(250, 250, 250);
  }

  .storybook-code {
    padding: 0.5rem;

    code { display: block; }
  }
}
