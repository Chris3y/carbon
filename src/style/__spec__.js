import buildPalette from './palette';
import palette from './palette';
import mix from './utils/mix';
import config from './color-config';



const assertCorrectColorMix = (config, paletteObject) => {
  Object.keys(config).forEach(col => {
    const match = /([A-Za-z]+)([\d]?[\d]?)/.exec(col);

    const func = match[1], weight = Number(match[2]);

    if (!paletteObject[func]) return 

    expect(paletteObject[func](weight)).toEqual(config[col]);
  })
}


describe('functional palette', () => {
  let colorConfig = {
    brilliantGreenBase: '00DC00',
    brilliantGreenShade20: '00B000',
    brilliantGreenTint50: '7FED7F',
    goldBase: 'FFB500',
    goldTint50: 'FFDA7F',
    errorRedBase: 'C7384F',
    errorRedShade20: '9F2C3F',
    genericGreenTint50: '7FCC7F',
    genericGreenTint30: '4CB74C',
    genericGreenShade15: '008200',
    genericGreenShade35: '006300',
    genericGreenShade55: '004400',
    productGreenTint50: '7FD1BA',
    productGreenTint30: '4CBE9F',
    productGreenShade21: '00805D',
    productGreenShade41: '006045',
    productGreenShade61: '003F2E',
    productBlueTint50: '7FBBE3',
    productBlueTint30: '4C9FD8',
    productBlueShade3: '0073C2',
    productBlueShade23: '005B9A',
    productBlueShade43: '004372',
    amethystTint50: 'AB95C1',
    amethystTint30: '8A6BA8',
    amethystTint10: '68418F',
    amethystBase: '582C83',
    amethystShade10: '4F2775',
    amethystShade30: '3D1E5B',
    slateTint95: 'F2F4F5',
    slateTint90: 'E5EAEC',
    slateTint80: 'CCD6DA',
    slateTint60: '99ADB6',
    slateTint40: '668491',
    slateTint20: '335B6D',
    slateShade60: '00141D'
  }

  it('produces the correct color mix', () => {
    assertCorrectColorMix(colorConfig, palette(config))
  })
})
