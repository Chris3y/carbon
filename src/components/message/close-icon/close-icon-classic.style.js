import { css } from 'styled-components';
import { THEMES } from '../../../style/themes';
import classicThemeColors from '../message-classic-theme-colors';

export default ({ theme, messageType }) => theme.name === THEMES.classic && css`
    background-color: ${classicThemeColors[messageType].backgroundColor};
`;
