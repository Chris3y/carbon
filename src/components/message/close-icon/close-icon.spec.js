import React from 'react';
import TestRenderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import CloseIcon from './close-icon.component';
import { CloseIconStyle, LinkStyle } from './close-icon.style';
import 'jest-styled-components';
import OptionsHelper from '../../../utils/helpers/options-helper/options-helper';
import classicTheme from '../../../style/themes/classic';

function render(props) {
  return TestRenderer.create(<CloseIconStyle { ...props } />);
}

describe('CloseIcon', () => {
  describe('when rendered', () => {
    it('should render correctly', () => {
      expect(shallow(<CloseIcon />)).toHaveLength(1);
    });
  });

  describe('when render with no additional props', () => {
    OptionsHelper.messages.forEach((messageType) => {
      it(`should match snapshot for ${messageType}`, () => {
        const wrapper = render({ messageType });
        expect(wrapper).toMatchSnapshot();
      });
    });
  });
});

describe('when in classic mode', () => {
  OptionsHelper.colors.forEach((messageType) => {
    describe(`when rendered as ${messageType}`, () => {
      it('should match the snapshot', () => {
        const wrapper = render({ messageType, theme: classicTheme });
        expect(wrapper).toMatchSnapshot();
      });

      describe('when transparent prop is set to true', () => {
        it('should apply white color background', () => {
          const wrapper = render({
            transparent: true,
            theme: classicTheme,
            messageType
          });
          expect(wrapper).toMatchSnapshot();
        });
      });
    });
  });
});

describe('LinkStyle', () => {
  it('applies peoper styling when focused', () => {
    const wrapper = TestRenderer.create(<LinkStyle />);
    expect(wrapper).toMatchSnapshot();
  });
});
