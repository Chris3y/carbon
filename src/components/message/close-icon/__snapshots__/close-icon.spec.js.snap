// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CloseIcon when render with no additional props should match snapshot for error 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: auto;
  margin-right: 15px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  border: none;
  color: #C7384F;
}

<div
  className="c0"
/>
`;

exports[`CloseIcon when render with no additional props should match snapshot for info 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: auto;
  margin-right: 15px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  border: none;
  color: #0073C2;
}

<div
  className="c0"
/>
`;

exports[`CloseIcon when render with no additional props should match snapshot for success 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: auto;
  margin-right: 15px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  border: none;
  color: #00B000;
}

<div
  className="c0"
/>
`;

exports[`CloseIcon when render with no additional props should match snapshot for warning 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: auto;
  margin-right: 15px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  border: none;
  color: #E96400;
}

<div
  className="c0"
/>
`;

exports[`LinkStyle applies peoper styling when focused 1`] = `
.c0 .carbon-link__content {
  display: none;
}

.c0 .carbon-link__icon {
  margin-right: 0;
}

.c0:focus {
  outline: none;
}

.c0:focus span span:before {
  outline: 2px solid #FFB500;
  outline-offset: 3px;
}

<a
  className="carbon-link__anchor c0"
  data-component="link"
  href={null}
  onKeyDown={[Function]}
  tabIndex="0"
  theme={
    Object {
      "colors": Object {
        "border": "#668491",
        "brand": "#00DC00",
        "error": "#C7384F",
        "focus": "#FFB500",
        "info": "#0073C2",
        "primary": "#26A826",
        "secondary": "#006300",
        "success": "#00B000",
        "tertiary": "#004400",
        "warning": "#E96400",
        "white": "#FFFFFF",
      },
      "disabled": Object {
        "border": "#CCD6DA",
        "disabled": "rgba(0,0,0,0.55)",
        "input": "#F2F4F5",
      },
      "name": "base",
      "shadows": Object {
        "cards": "0 3px 3px 0 rgba(0,20,29,0.2), 0 2px 4px 0 rgba(0,20,29,0.15)",
        "depth1": "0 5px 5px 0 rgba(0,20,29,0.2), 0 10px 10px 0 rgba(0,20,29,0.1)",
        "depth2": "0 10px 20px 0 rgba(0,20,29,0.2), 0 20px 40px 0 rgba(0,20,29,0.1)",
        "depth3": "0 10px 30px 0 rgba(0,20,29,0.1), 0 30px 60px 0 rgba(0,20,29,0.1)",
        "depth4": "0 10px 40px 0 rgba(0,20,29,0.04), 0 50px 80px 0 rgba(0,20,29,0.1)",
      },
      "text": Object {
        "color": "rgba(0,0,0,0.9)",
        "placeholder": "rgba(0,0,0,0.3)",
        "size": "14px",
      },
    }
  }
>
  <span>
    <span
      className="carbon-link__content"
    />
  </span>
</a>
`;

exports[`when in classic mode when rendered as default should match the snapshot 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: auto;
  margin-right: 15px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  border: none;
  background-color: #335c6d;
}

<div
  className="c0"
/>
`;

exports[`when in classic mode when rendered as default when transparent prop is set to true should apply white color background 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: auto;
  margin-right: 15px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  border: none;
  background-color: #335c6d;
  background-color: #FFFFFF;
}

<div
  className="c0"
/>
`;

exports[`when in classic mode when rendered as error should match the snapshot 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: auto;
  margin-right: 15px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  border: none;
  color: #C7384F;
  background-color: #fdf5f5;
}

<div
  className="c0"
/>
`;

exports[`when in classic mode when rendered as error when transparent prop is set to true should apply white color background 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: auto;
  margin-right: 15px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  border: none;
  color: #C7384F;
  background-color: #fdf5f5;
  background-color: #FFFFFF;
}

<div
  className="c0"
/>
`;

exports[`when in classic mode when rendered as help should match the snapshot 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: auto;
  margin-right: 15px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  border: none;
  background-color: #fffbf2;
}

<div
  className="c0"
/>
`;

exports[`when in classic mode when rendered as help when transparent prop is set to true should apply white color background 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: auto;
  margin-right: 15px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  border: none;
  background-color: #fffbf2;
  background-color: #FFFFFF;
}

<div
  className="c0"
/>
`;

exports[`when in classic mode when rendered as info should match the snapshot 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: auto;
  margin-right: 15px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  border: none;
  color: #0073C2;
  background-color: #f3f8fe;
}

<div
  className="c0"
/>
`;

exports[`when in classic mode when rendered as info when transparent prop is set to true should apply white color background 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: auto;
  margin-right: 15px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  border: none;
  color: #0073C2;
  background-color: #f3f8fe;
  background-color: #FFFFFF;
}

<div
  className="c0"
/>
`;

exports[`when in classic mode when rendered as maintenance should match the snapshot 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: auto;
  margin-right: 15px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  border: none;
  background-color: #fff8f2;
}

<div
  className="c0"
/>
`;

exports[`when in classic mode when rendered as maintenance when transparent prop is set to true should apply white color background 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: auto;
  margin-right: 15px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  border: none;
  background-color: #fff8f2;
  background-color: #FFFFFF;
}

<div
  className="c0"
/>
`;

exports[`when in classic mode when rendered as new should match the snapshot 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: auto;
  margin-right: 15px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  border: none;
  background-color: #f7f5fa;
}

<div
  className="c0"
/>
`;

exports[`when in classic mode when rendered as new when transparent prop is set to true should apply white color background 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: auto;
  margin-right: 15px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  border: none;
  background-color: #f7f5fa;
  background-color: #FFFFFF;
}

<div
  className="c0"
/>
`;

exports[`when in classic mode when rendered as success should match the snapshot 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: auto;
  margin-right: 15px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  border: none;
  color: #00B000;
  background-color: #f6fbf6;
}

<div
  className="c0"
/>
`;

exports[`when in classic mode when rendered as success when transparent prop is set to true should apply white color background 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: auto;
  margin-right: 15px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  border: none;
  color: #00B000;
  background-color: #f6fbf6;
  background-color: #FFFFFF;
}

<div
  className="c0"
/>
`;

exports[`when in classic mode when rendered as warning should match the snapshot 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: auto;
  margin-right: 15px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  border: none;
  color: #E96400;
  background-color: #fff8f2;
}

<div
  className="c0"
/>
`;

exports[`when in classic mode when rendered as warning when transparent prop is set to true should apply white color background 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: auto;
  margin-right: 15px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  border: none;
  color: #E96400;
  background-color: #fff8f2;
  background-color: #FFFFFF;
}

<div
  className="c0"
/>
`;
